loopchain/channel/channel_inner_service.py

```diff
@@ -601,6 +602,32 @@ async def get_status(self):
     async def make_essential_backup(self, block_height):
         return await BackupManager().make_backup(self._blockchain, block_height)
 
+    @message_queue_task(priority=255)
+    async def block_height_request(self, request_from: str):
+        """Handle BlockHeightRequest.
+
+        Responses last height of this node.
+        """
+        last_block = self._channel_service.block_manager.blockchain.last_block
+        height: int = last_block.header.height
+        response_type = loopchain_pb2.HeightResponse(
```

Comment:
What does `type` in `response_type` mean ? 
I guess only the `HeightResponse` object can be the parameter to the `BlockHeightResponse` message, and it is just response data. 
That's why I guess `type` is an unnecessary word.

Reply:
Edited!